lib/content-services/content-node-selector/content-node-dialog.service.ts

```diff
@@ -72,33 +72,40 @@ export class ContentNodeDialogService {
     /** @param action Name of the action (eg, "Copy" or "Move") to show in the title */
     /** @param contentEntry Item to be copied or moved */
     /** @param permission Permission for the operation */
-    openCopyMoveDialog(action: string, contentEntry: MinimalNodeEntryEntity, permission?: string): Observable<MinimalNodeEntryEntity[]> {
+    openMoveDialog(action: string, contentEntry: MinimalNodeEntryEntity, permission?: string): Observable<MinimalNodeEntryEntity[]> {
```

Comment:
Should this be a breaking change? The public method gets renamed.

Reply:
nope

Reply:
the component input and output and name are our contracts

Reply:
what is someone is using this api already?

Reply:
But this is a service

Reply:
On ACA side we do not have this problem because we have this method:
 ` private isActionAllowed(action: string, node: MinimalNodeEntryEntity, permission?: string): boolean {
        if (action === 'copy') {
            return true;
        }
        return this.contentService.hasPermission(node, permission);
    }`
Maybe it would be good to consider such an approach so that there are not too many code changes introduced. What do you think?

Reply:
I think it's ok, so i will close this one and create a new one adding the new enum and condition to the method :)